shotover-proxy/Cargo.toml

```diff
@@ -54,7 +54,8 @@ halfbrown = "0.1.11"
 
 # Transform dependencies
 redis-protocol = "3.0.1"
-cassandra-protocol = { git = "https://github.com/krojew/cdrs-tokio" }
+#cassandra-protocol = { git = "https://github.com/krojew/cdrs-tokio" }
+cassandra-protocol = { git = "https://github.com/conorbros/cdrs-tokio", branch = "typed-collections" }
```

Comment:
is this an accident to?

Reply:
No, I had to put some changes into cdrs-tokio to fix the BigInt issue.

Reply:
ah ok, I was confused because it had the same branch name.
Dont forget to make a PR for it.

Reply:
Yep, just wanted to confirm it's the right approach before I made a PR.

Reply:
https://github.com/krojew/cdrs-tokio/pull/73